### PR TITLE
Bookstore: Initialize components

### PR DESCRIPTION
**Pull request by arthurborgesdev**
# In this Pull Request I implemented:

- Created a blank repository and initialize it with create-react-app
- Add ESLinter and Stylelint linters and fix errors to run the application
- Add Redux to the project
- Add "feature folder structure" approach to contain the Redux files
- Add re-usable components for book page
- Assembled re-usable blocks
- Add minimum styling to make the app reassemble the provided layout
- Add routes and customize the links for books and categories
- Add README and screenshot to the project
- Fix ESLint and Stylelint errors

### CODE REVIEW

File: src/components/BookNew.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BookNew = () => (
+  <>
+    <h2>ADD NEW BOOK</h2>
+    <div className="add-book-form">
```
Comment (Davidosky007) [inline src/components/BookNew.js:6]: * I get this error on the console on page load kindly fix it 🙏 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @arthurborgesdev 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/BookNew.js (hunk 1 above)
Comment (arthurborgesdev) [reply src/components/BookNew.js:6]: Fixed!

**Comment by Davidosky007** [comment]
## Hi @arthurborgesdev 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
##Congratulations! 🎉 
nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**